spec/site_prism/page_objects/page/organisations/course_details.rb

```diff
@@ -32,6 +32,7 @@ class CourseDetails < CourseBase
         element :level, "[data-qa=course__level]"
         element :entry_requirements, "[data-qa=course__entry_requirements]"
         element :allocations_info, "[data-qa=course__allocations_info]"
+        element :changing_your_basic_details, "[data-qa=course__changing_your_basic_details]"
```

Comment:
sanity check @aliuk2012 we dropping `data-qa` right?

Reply:
Yes are are in the process of dropping `data-qa`